conf/svcomp22.json

```diff
@@ -1,12 +1,12 @@
 {
   "ana": {
+    "autoselect": true,
```

Comment:
This conf file reflects our configuration at SV-COMP 2022 and thus should not be changed.